implementations/rust/ockam/ockam_command/src/global_args.rs

```diff
@@ -36,85 +34,55 @@ pub struct GlobalArgs {
     pub verbose: u8,
 
     /// Disable colors in output
-    #[arg(global = true, long, default_value_t = no_color_default_value())]
+    #[arg(global = true, long, env = "NO_COLOR")]
     pub no_color: bool,
 
     /// Disable tty functionality, like interactive prompts.
-    #[arg(global = true, long, default_value_t = no_input_default_value())]
+    #[arg(global = true, long, env = "NO_INPUT")]
     pub no_input: bool,
 
     /// Specifies the output format of the command. Defaults to 'plain' if not explicitly set.
     /// The 'plain' format is a piece of plain text, the content of which may change based on whether
     /// the stdout is a tty or not. For instance, if stdout is redirected to a file, the output
     /// is usually an identifier that can be used as input for other commands. If stdout is a tty,
     /// the output will contain human-readable information about the command execution.
-    /// The 'json' format can be customized with the `--jq` and `--pretty` options.
+    /// The 'json' format can be customized with the `--jq` and `--compact-output` options.
     #[arg(global = true, long = "output", value_enum)]
-    pub output_format: Option<OutputFormatArg>,
+    pub(crate) output_format: Option<OutputFormatArg>,
 
     /// jq query to apply to the JSON output of the command
     #[arg(global = true, long = "jq")]
-    pub jq_query: Option<String>,
+    jq_query: Option<String>,
 
-    /// Pretty print the JSON output of the command
+    /// Compact the JSON output of the command
     #[arg(global = true, long)]
-    pub pretty: bool,
+    compact_output: bool,
```

Comment:
Should we add an alias for `--pretty` to keep back-compatibility?

Reply:
Yeah good idea, but it will have to be a separate argument. I'll add it in a bit

Reply:
ready now!